src/main/java/Tommi.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class Tommi {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        printIntroMessage();
+        readInputStrings(scanner);
+        printExitMessage();
+    }
+
+    private static void readInputStrings(Scanner scanner) {
+        String input = scanner.nextLine();
+        while (!input.equals("bye")) {
+            InputHandler.processInputCases(input);
+            input = scanner.nextLine();
+        }
+    }
+
+    private static void printIntroMessage() {
+        String intro = """
+                 ______                  \s
+                /_  __/__  __ _  __ _  (_)
+                 / / / _ \\/  ' \\/  ' \\/ /\s
+                /_/  \\___/_/_/_/_/_/_/_/ \s
+                ____________________________________________________________
+                Hello! I'm Tommi!
+                What can I do for you?
+                ____________________________________________________________
+                """;
+        System.out.println(intro);
+    }
```

Comment:
I appreciate the effort gone to customise your logo, and abstract it to a function for later use!